doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -9,22 +9,93 @@
 * [3.3 Cloud Infrastructure Software Profile](#3.3)
 * [3.4 Cloud Infrastructure Hardware Profile](#3.4)
 
+
+
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-> Describe the purpose of this chapter - to describe the requirements on any labs that are used to deploy the RI.
+The purpose of this chapter is to list and describe the requirements on labs used to deploy and run the RI2. Overall, the RI2 serves two purposes which guide the definition of requirements in this chapter:
+
+1. RI2 serves as a baseline and prooving ground for developing the [Reference Conformance 2 (RC2) specification](../../ref_cert/RC2) as well as the Cloud Native OVP program (LINK), and
+1. RI2 enables software vendors of CNFs and cloud container platforms to deploy a reference implementation in their development environment to develop and validate against.
+
+As a result of those two main use cases, RI2 must be deployable in different lab environments:
+
+1. internal development labs of software vendors, operators and 3rd party test labs,
+1. public community labs (e.g. OPNFV),
+1. resources hosted by cloud infrastructure providers (e.g. Packet.com)
+
+In addition, lab environments must meet specific requirements on hardware resources (e.g. compute and storage), capabilities (e.g. support for SR-IOV), and connectivity (e.g., number of NICs) to run the RI2. The following sections describe the resulting requirements in detail.
+
+
 
 <a name="3.2"></a>
 ## 3.2 Access and Connectivity
 
-> Describe the access and connectivity requirements for any lab used to deploy an RI.
+This section describes the access and connectivity requirements for labs used to deploy an RI.
+
+
+<a name="3.2.1"></a>
+### 3.2.1 Lab Access
+
+The requirements on and means of accessing a given lab differ per hosting organization and use case:
+
+* **Community labs**, such as those hosted by OPNFV, must provide remote access from the Internet to allow usage of the lab resources by community members. Remote access shall be managed and secured by means of VPN or SSH gateways. Both the public and out-of-band management networks of the lab shall be accessible from the VPN / SSH gateway host.
+
+* Access to lab resources hosted by **public cloud providers** is managed by the cloud provider. If a public cloud provider is chosen to host community-based development resources, access to those resources shall be available to all community members by means of a application process. 
```

Comment:
Typo: "... by means of a application process."  ---> "... by means of an application process."

Access to community labs irrespective if hosted by OPNFV, 3rd Party or public cloud would require a process to obtain access. So the statement (last sentence) within the "public cloud providers" paragraph needs to be under the "community labs" paragraph.

Reply:
@pgoyal01: I moved the respective statement to a separate section because, as you pointed out, the defining criteria for requiring access is that a given infrastructure is used for a *community activity*, irrespective of the type of lab or infrastructure.